src/views/partials/add-email.js

```diff
@@ -2,16 +2,24 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+import { getMessage } from '../../utils/fluent.js'
+
 export const addEmail = data => `
 <header>
-  <h1>${data.something} Add another email address</h1>
   <button class='close'></button>
+  <img src='/images/dialog-email-clouds.svg'>
+  <h2>${getMessage('add-email-add-another-heading')}</h2>
 </header>
-<section>
-  <p>Your account includes monitoring of up to 5 email addresses. Add a new email address to see if it’s been involved in a breach.</p>
-  <form>
-    <label>Email address<input type='email'></label>
-    <button class='primary' type='submit'>Send verification link</button>
-  </form>
-</section>
+<form>
+  <p>${getMessage('add-email-your-account-includes', { total: data.emailLimit })}</p>
+  <input type='hidden' name='csrf-token' value='${data.csrfToken}'>
+  <label class='text-field'>
+    <span>${getMessage('add-email-address-input-label')}</span>
+    <input type='email' name='email-address' required>
+  </label>
+  <button class='primary' type='submit' name='email-submit'>${getMessage('add-email-send-verification-button')}</button>
+</form>
+<template data-success>
+  <p class='add-email-success'>${getMessage('add-email-verify-the-link', { email: '<b class="current-email"></b>', 'settings-href': 'href="/user/settings"' })}</p>
```

Comment:
Nit (non-blocking): This line is getting a bit long.

Reply:
fixed via https://github.com/mozilla/blurts-server/pull/2830/commits/11ec0ba60e03dabff53b64d089d7f932778540bc